File: src/main/java/org/funivan/intellij/FastCoddy/LanguageProcessor/CodeExpandInterface.java
package org.funivan.intellij.FastCoddy.LanguageProcessor;

import com.intellij.openapi.actionSystem.AnActionEvent;
import org.funivan.intellij.FastCoddy.CodeBuilders.CodeTemplate;

/**
 * @author Ivan Scherbak <dev@funivan>
 */
public interface CodeExpandInterface {
    
    CodeTemplate getCode(AnActionEvent anActionEvent);

    
}

File: src/main/kotlin/org/funivan/intellij/FastCoddy/LanguageProcessor/CodeExpandInterface.kt
package org.funivan.intellij.FastCoddy.LanguageProcessor

import com.intellij.openapi.actionSystem.AnActionEvent
import org.funivan.intellij.FastCoddy.CodeBuilders.CodeTemplate

/**
 * @author Ivan Shcherbak <alotofall@gmail.com>
 */
interface CodeExpandInterface {
    fun getCode(anActionEvent: AnActionEvent): CodeTemplate?
}
